test/expr/expressions.swift

```diff
@@ -502,6 +502,10 @@ func stringliterals(_ d: [String: Int]) {
   // expected-error @+1 {{unterminated string literal}}
   var x2 : () = ("hello" + "
   ;
+
+  // expected-note @+2 {{Add an ending quote to string literal}}
```

Comment:
We also want to check for the fix-it 
```suggestion
  // expected-note @+2 {{Add an ending quote to string literal}} {{22-22="}}
```

Reply:
This has made me realize a fault in my own code which is that the end quotes for multiline strings is different than the one for non-multiline strings.. are there any methods for detecting if a string is multi line?

Reply:
Never mind my last comment, there’s a local variable for that in the same function.. I’ll commit the changes in a sec :)